src/app/docs/tokens/tokens-docs.component.ts

```diff
@@ -1,9 +1,62 @@
 import {
   Component
 } from '@angular/core';
+import { SkyDocsDemoControlPanelChange } from '@skyux/docs-tools';
+
+import {
+  SkyToken
+} from '../../public/public_api';
 
 @Component({
   selector: 'app-tokens-docs',
   templateUrl: './tokens-docs.component.html'
 })
-export class TokensDocsComponent { }
+export class TokensDocsComponent {
+
+  public colors: SkyToken[];
+
+  private defaultColors = [
+    { name: 'Red' },
+    { name: 'Black' },
+    { name: 'Blue' },
+    { name: 'Brown' },
+    { name: 'Green' },
+    { name: 'Orange' },
+    { name: 'Pink' },
+    { name: 'Purple' },
+    { name: 'Turquoise' },
+    { name: 'White' },
+    { name: 'Yellow' }
+  ];
+
+  public demoSettings: any = {};
+
+  constructor() {
+    this.colors = this.parseTokens(this.defaultColors);
+  }
+
+  public onDemoSelectionChange(change: SkyDocsDemoControlPanelChange): void {
+    if (change.disabled !== undefined) {
+      this.demoSettings.disabled = change.disabled;
+    }
+    if (change.dismissible !== undefined) {
+      this.demoSettings.dismissible = change.dismissible;
+    }
+    if (change.focusable !== undefined) {
+      this.demoSettings.focusable = change.focusable;
+    }
+  }
+
+  public onDemoReset(): void {
+    this.colors = this.parseTokens(this.defaultColors);
+  }
+
+  private parseTokens(data: any[]): SkyToken[] {
```

Comment:
This is my mistake, but "parse" only deals with converting `string` data into something else, so it doesn't make much sense to use here (I learned this recently, after adding that word to a lot of our docs-tools methods).

I'd change it to `getTokens` or `convertToTokens`.